packages/esm-service-queues-app/src/metrics/metrics-cards/metrics-card.component.tsx

```diff
@@ -56,9 +60,17 @@ const MetricsCard: React.FC<MetricsCardProps> = ({
             </div>
           )}
         </div>
-        <div>
-          <label className={styles.totalsLabel}>{label}</label>
-          <p className={styles.totalsValue}>{value}</p>
+        <div className={styles.metricsContainer}>
+          <div className={styles.metricItem}>
+            <label className={styles.totalsLabel}>{label}</label>
+            <p className={styles.totalsValue}>{value}</p>
+          </div>
+          {showUrgent && (
+            <div className={styles.metricItem}>
+              <label className={styles.urgentLabel}>{t('urgent', 'Urgent')}</label>
```

Comment:
I don't see the corresponding class for `.urgentLabel` in the CSS file. Is it meant to extend `.totalsLabel`?

```scss
.urgentLabel {
  @extend .totalsLabel;
}
```